series.tex

```diff
@@ -0,0 +1,79 @@
+% Series expansion (Differentiate between the two approaches being used)
+\subsubsection{Series Expansion}
+
+SymPy is able to calculate the symbolic series expansion of an arbitrary series
+or expression involving elementary and special functions and multiple
+variables. For this it has two different implementations.
+
+The first approach stores a series as a core object. Each function has its
+specific implementation of its expansion. It is able to evaluate the Puiseux
+(i.e. rational exponents are allowed in series) expansion about any point.
+
+\begin{verbatim}
+>>> from sympy import *
+>>> x, y = symbols('x, y')
+>>> series(sin(x + y) + cos(x*y), x, 0, 2)
+1 + sin(y) + x*cos(y) + O(x**2)
+\end{verbatim}
+
+The newer and much faster approach called Ring Series makes use of the
+observation that a truncated Taylor series, is in fact a polynomial.
+Ring-series uses the efficient representation and operations of sparse
+polynomials. The choice of sparse polynomials is deliberate as it performs
+well in a wider range of cases than a dense representation. It gives the user
+the freedom to choose the type of coefficients he wants to have in his series,
+allowing the use of faster operations on certain types.
+
+For this, several low level methods for expansion of elementary functions are
+implemented using semi-numerical algorithms. All these support Puiseux seris
+expansion.
+
+\begin{verbatim}
+>>> from sympy.polys.ring_series import rs_sin
+>>> R, x = ring('x', QQ)
+>>> rs_sin(x**2 + x, x, 5)
+-1/2*x**4 - 1/6*x**3 + x**2 + x
+\end{verbatim}
+
+The function \texttt{rs\_series} makes use of these elementary functions to
+intelligently expand an arbitrary SymPy expression. Currently it supports
+expansion only about 0 and is under active development. Typical speedup over
+the older SymPy implementation is in the range 20-100 times with larger
```

Comment:
@isuruf How do you suggest I add a benchmark? Should I post a link to a wiki page?